reader_test.go

```diff
@@ -601,6 +601,31 @@ func TestMediaPlaylistWithOATCLSSCTE35Tag(t *testing.T) {
 	}
 }
 
+func TestMediaPlaylistWithCueSCTE35Tag(t *testing.T) {
+	f, err := os.Open("sample-playlists/media-playlist-with-cue-scte35.m3u8")
+	if err != nil {
+		t.Fatal(err)
+	}
+	p, _, err := DecodeFrom(bufio.NewReader(f), true)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Same as my other comment. ioutil.ReadFile would be simpler.

Reply:
Replace os.Open with ioutil.ReadFile in:  https://github.com/zencoder/m3u8/pull/8/commits/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0